src/components/Transactions/BlockExplorer.tsx

```diff
@@ -0,0 +1,37 @@
+import { ExportOutlined } from '@ant-design/icons';
+
+import { shortAddress } from '../../utils';
+import { ChainId, NetworkScanUrl, BlockchainExplorer } from '../../constants';
+interface IProps {
+  address: string;
+  chars?: number;
+  chainId: number;
+  isCheckingTx: boolean;
+  displayedText?: any;
```

Comment:
remove this, and instead use `children`, that way you can do stuff like:
```jsx
<BlockExplorerLink ...>
   <ExportOutlined />
   some text
</BlockExplorerLink>
```
And this also allows you to remove that super ugly `if (displayedText === 'ScanMenuItem') {`

If no `children` passed, then you can default to `shortAddress`